provisioning/bootstrap.sh

```diff
@@ -36,13 +43,19 @@ DB_PASSWORD = ${DB_PASSWORD}
 EMAIL_BACKEND = django.core.mail.backends.console.EmailBackend
 EOF
 
-sudo -H -u vagrant pip3 install --user -r /vagrant/requirements.txt scikit-learn
+sudo -H -u vagrant pip3 install virtualenv
+sudo -H -u vagrant /home/vagrant/.local/bin/virtualenv --python=python3 $VIRTUALENV_PATH
```

Comment:
I see now why it's failing, in my head, the line before was `pip3 install --user virtualenv` which would install into a `.local` folder.
But `pip3 install virtualenv` is most likely going to fail due to PermissionErrors.

`pipenv` is a great idea!